Default.sublime-commands

```diff
@@ -81,6 +81,14 @@
         "caption": "LSP: Goto Declaration",
         "command": "lsp_symbol_declaration",
     },
+    {
+        "caption": "LSP: Goto Diagnostic",
```

Comment:
The name doesn't make it clear that it only shows diagnostics for the current file. Maybe `LSP: Goto Diagnostic in File`?

Reply:
Yes, I agree. I took it from "Goto Symbol..." and "Goto Symbol in Project...", but adding "for File' makes it clear immediately.

Reply:
Hm, I just tried it and I wouldn't do this. It doesn't feel right. My understanding is the non-qualified "Goto (whatever)" implies the active view as context.

With "Goto Diagnostic for File" there is immediately the question "which file?" and I would expect a file chooser, but this is exactly what the other command does.